source/whats-new.txt

```diff
@@ -42,6 +42,18 @@ The v4.0 driver release includes the following new features:
 - Adds a getter method for the ``start_at_operation_time`` field of a 
   ``mongocxx::options::change_stream`` instance.
 
+The release includes the following bug:
+
+- CMake might incorrectly set the API version (``BUILD_VERSION``) for an 
+  auto-downloaded C driver to ``0.0.0`` or to the same value as the C++ driver's 
+  API version. CMake will only auto-download the C driver with the C++ driver 
+  if it cannot find an existing C driver installation. This ``BUILD_VERSION`` 
+  error will be fixed in an upcoming patch release. 
```

Comment:
Consider tweaking wording to:

> CMake might incorrectly set the API version for an auto-downloaded C Driver to `0.0.0` when the CMake project is configured more than once or to the same value as the C++ Driver's API version when `BUILD_VERSION` is explicitly set during initial configuration. CMake will only auto-download the C Driver if it cannot find an existing C Driver installation via `find_package()`. This bug will be fixed in an upcoming patch release.

This describes the conditions required to encounter this bug (reconfiguration or explicit `BUILD_VERSION`) as well as a more accurate description of how an existing C Driver installation is found (via `find_package()`).

Reply:
Good edit, thanks!